### DISCUSSION TITLE
GPS location, lattitude and longitude, open popup or screen and show google maps with these gps location

### DISCUSSION THREAD

**Discussion by killerbeat12** [frangoteam/FUXA#1224] [Q&A]
i receive from my device the lattitude and longitude, is it possible to write a script or something to open google maps in a new popup or screen with these values?

**Comment by unocelli** [comment]
Hi, there is a similar request #174

**Comment by killerbeat12** [comment]
Thanks for letting me know, so also for this request still nothing is in place right? i was thinking about something simple, just read the 2 values (lattitude and longitude) in script and then at them to a URL string and open this in new popup
For example: POPUP URL = "https://www.google.com/maps/@" & [Lattitude] & "," & [Longitude ]& ",14z?entry=ttu"
